.eslintrc.yml

```diff
@@ -100,9 +99,7 @@ rules:
   '@typescript-eslint/no-explicit-any': off
   '@typescript-eslint/no-inferrable-types': off
   '@typescript-eslint/no-empty-function': off
-
-  # Babel
-  babel/no-invalid-this: error
```

Comment:
I don't know what was happening here but as soon as I bumped one of the `@typescript-eslint` packages this started reporting errors all over the place so I had to disable it. I can't remember a single time this rule has saved us so I'm not concerned about it going away.